cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -457,6 +467,7 @@ def _act_on_(
         return NotImplemented
 
 
+@dataclass(frozen=True, init=False, eq=False, repr=False)
```

Comment:
What does making this a dataclass actually do here with all these options set to False?

Reply:
makes the class frozen while not overriding any method. the default parameters will create an `__init__`, `__eq__` and `__repr__` but we already implement these.